app/views/reservations/new.html.haml

```diff
@@ -21,8 +21,8 @@
 
 .wysiwyg= @instrument.description
 
-- unless @instrument.online?
-  %p.alert.alert-danger= t(".offline_notice")
+- if @instrument.offline? || @instrument.offline_via_schedule_share?
```

Comment:
Is the `offline?` check redundant? This instrument is included in `schedule.products`, so it's status should get checked by `offline_via_schedule_share?`. 

Also, that name seems a little awkward. `offline_via_shared_schedule`?